crates/sui-core/src/authority.rs

```diff
@@ -1221,6 +1242,7 @@ impl AuthorityState {
         expected_effects_digest: Option<TransactionEffectsDigest>,
         epoch_store: &Arc<AuthorityPerEpochStore>,
     ) -> SuiResult<(TransactionEffects, Option<ExecutionError>)> {
+        let process_certificate_start_time = tokio::time::Instant::now();
```

Comment:
I think you want to move this down a bit so that it doesn't include the already-executed case

Reply:
Never mind. This won't be reported if early return anyway.